app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+from datetime import date
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        count = 0
+        if "vaccine" not in visitor:
+            count += 1
+            raise NotVaccinatedError("The client does not have a vaccine")

```

Comment:
The error message for `NotVaccinatedError` should be more descriptive. According to the checklist, use descriptive error messages. Consider providing more context about the error.